### ISSUE TITLE
False positive: label rule fails for input elements with explicit hidden label despite having accessible label by other means

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1338]
label rule fails for input elements with explicit hidden label despite having accessible label using aria-labelledby

- Link to live demo: https://codepen.io/iamrafan/pen/BMjOEB?editors=1000

-  AT test results:

  | AT/Browser | Case-1 | Case-2
-- | -- | -- | --
1 | Edge/Narrator | Name-case-2   editing | Name-case-3   editing
2 | Edge/JAWS | Name-case-2 Edit | Name-case-3 Edit
3 | Edge/NVDA | Name-case-2  edit | Name-case-3  edit
4 | Chrome/NVDA | Name-case-2  edit | Name-case-3  edit
5 | Chrome/JAWS | Name-case-2  edit | Name-case-3  edit
6 | IE/JAWS | Name-case-2  edit | Name-case-3  edit
7 | IE/NVDA | Name-case-2  edit | Name-case-3  edit
8 | Firefox/NVDA | Name-case-2  edit    has auto complete | Name-case-3  edit    has auto complete

<pre><code>axe-core version: 3.1.2
axe-Coconut version: 3.7.0</code></pre>

**Comment by WilcoFiers** [comment]
Hey Guru, thanks for reporting. This is a known issue. We have a fix for it in the canary version of axe-core already. You should see this resolved in the 3.2.0 release. I'm going to close this as this issue seems resolved.

**Comment by iamrafan** [comment]
sure, thank you.
